app/views/documents/form_partials/_validate_document.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with model: [@planning_application, @document], local: true, class: "document", builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+  <div class="govuk-form-group" id="validate-document">
+    <% if form.object.errors.any? %>
+      <% form.object.errors.each do |error| %>
+        <span class="govuk-error-message">
+        <span class="govuk-visually-hidden">Error:</span><%= error.type %></span>
+      <% end %>
+    <% end %>
+    <%= form.govuk_radio_buttons_fieldset(:validated, legend: { size: 's', text: 'Is the document valid?' }) do %>
+      <%= form.govuk_radio_button :validated, 'true', label: { text: 'Yes'} %>
+      <%= form.govuk_radio_button :validated, 'false', label: { text: 'No' } do  %>
+        <%= form.govuk_text_area :invalidated_document_reason, label: { text: '' }, hint: { text: 'Describe in full why the document is invalid. This will be sent to the applicant.'}, rows: 3 %>
```

Comment:
Andrew reminded me last week that it's better to always have text for labels even if you have to attach a class to visually hide the text if you don't want it to display. This would also help with the Capybara test, as you are not then having to use a selector to identify the input field

Reply:
changed, thank you!